src/components/KCodeBlock/KCodeBlock.vue

```diff
@@ -166,9 +160,7 @@
           class="k-next-match-button"
           data-testid="k-code-block-next-match-button"
           :disabled="matchingLineNumbers.length === 0 || isFilterMode"
-          :is-rounded="false"
           size="small"
-          title="Next match (F3)"
           type="button"
           @click="jumpToNextMatch"
         >
```

Comment:
**Issue**: The previous and next match buttons in particular don’t look right.

![image](https://github.com/Kong/kongponents/assets/5774638/8e4cdc3b-2d11-4a86-922f-65350e851507)

In their enabled state, they have a white text color on a light grey background that’s barely visible.

Perhaps this would be a good time to revisit the custom button styles that were added to this component over time (though I’m not sure if there is perhaps the intention to do so when working on KCodeBlock specifically). For example, removing the background color override on `.k-code-block.theme-light .k-button:not(.increase-specificity)` produces this look:

![image](https://github.com/Kong/kongponents/assets/5774638/e0f8427f-0449-4f54-b5bd-ddb55f9a7491)

I think that looks better.

Reply:
Yes, for now I will fix this to make it look okay, until we get to KCodeBlock when we will reskin the component to match the look and feel provided by design.

Reply:
The title attr should be left in place here as well